website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+
+`eksctl` also handles interconnecting Kubernetes RBAC and AWS IAM. For example, associating the `karpenter` service account with an IAM role for creating instances. 
+
+```
+eksctl create iamserviceaccount \
+  --cluster $CLUSTER_NAME --name karpenter --namespace karpenter \
+  --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/KarpenterControllerPolicy-$CLUSTER_NAME \
+  --approve
+```
+
+```
+eksctl create iamidentitymapping \
+  --username system:node:{{EC2PrivateDNSName}} \
+  --cluster  ${CLUSTER_NAME} \
+  --arn arn:aws:iam::${AWS_ACCOUNT_ID}:role/KarpenterNodeRole-${CLUSTER_NAME} \
+  --group system:bootstrappers \
+  --group system:nodes
+```
+
+
+### Launch Template
+
+A launch template is an AWS resource that bundles every configuration value needed to properly launch a new instance, such as base image and storage.
+
+Karpenter will create a launch template suitable for the environment described in the getting started guide (EKS, IRSA, etc) if none is specified.
+
+To the extent possible, nodes should be customized using Karpenter provisioner CRD fields. However, many environments require using a different base image (instead of the EKS optimized flavor of Amazon Linux 2), such as to incorporate a standardized security daemon at the node level. 
+
+Review the [detailed instructions](../cloud-providers/aws/launch-templates/) for creating a custom launch template. 
+
+## Deploy Karpenter
+
+The yaml files comprising the helm chart are suitable for production environments. They may be lightly modified for direct use, for example storage in an internal GitOps cluster repo (without referencing an external chart).
+
+The general structure of the helm chart and described resources:
+
+- [Helm Chart](https://github.com/awslabs/karpenter/tree/main/charts/karpenter)
+    - `Provisioner` CRD
+    - `Karpenter` Service Account
+    - Controller Deployment
+        - Karpenter Controller Image from ECR
+        - RBAC for Karpenter Service Account
+    - Webhook Deployment
+        - Karpenter Webhook Image from ECR
+        - RBAC for Karpenter Service Account
+        - Webhooks (Mutating, Validating)
+    - Default Provisioner
+    - Logging Configuration
+
+Review [`values.yaml`](https://github.com/awslabs/karpenter/blob/main/charts/karpenter/values.yaml) for helm chart configuration options.
+
+Some important fields are:
+- serviceAccount.create
+- serviceAccount.name
+- controller.clusterName
+- controller.clusterEndpoint
+
+## Mutations of Kubernetes Behavior
+Note: This section is an incomplete draft.
+
+- `k delete node`
+    - Deleting a karpenter managed node triggers deprovisioning of the cloud instance.
+- High amount of pods on a single node
+    - Karpenter often decides to use a large instance, and pack many pods onto the instance. 
+    - This may increase the load on *per node* daemonsets

```

Comment:
It also can reduce the load on the Kubernetes API and may need VPC CNI settings (eg prefix delegation). Do we have a doc somewhere we can reference with trade-offs and considerations?

Reply:
added those additional points. i'll take a look for existing documents.